charts/civil-ccd/values.template.yaml

```diff
@@ -1,6 +1,32 @@
 civil-ccd-stack:
   enabled: true
 
+postgresql:
```

Comment:
this should be part of another depdent chart.
e.g.
```
civil-general-applications:
    java:
        postgres:
```

example: https://github.com/hmcts/civil-camunda-bpmn-definition/pull/443/files#diff-9830039803a3e02266afb0fdd1d7cbfc9b60018e8f16a46dacac4f3f66924d53R73